README.md

```diff
@@ -74,10 +74,20 @@ $transfer  = $api->transfer->fetch($transferId)->edit($options); // Edit a trans
 $reversal  = $api->transfer->fetch($transferId)->reverse(); // Reverse a transfer
 
 // Payment Links
-$links = $api->invoice->all();
-$link  = $api->invoice->fetch('inv_00000000000001');
-$link  = $api->invoice->create(arary('type' => 'link', 'amount' => 500, 'description' => 'For XYZ purpose', 'customer' => array('email' => 'test@test.test')));
+$links = $api->payment_link->all(); // fetch all payment links
+
+$link  = $api->payment_link->fetch('plink_GiwM9xbIZqbkJp'); // fetch payment link with id
+
+$data = json_encode(
```

Comment:
@neera11, why we need to pass data as JSON for this entity only?

Reply:
yep..should be array. @neera11